include/xtensor/xbroadcast.hpp

```diff
@@ -94,6 +94,37 @@ namespace xt
         using size_type = typename xexpression_type::size_type;
     };
 
+    /*****************************
+     * linear_begin / linear_end *
+     *****************************/
+
+    namespace detail
+    {
+        template <class CT, class X>
+        constexpr auto linear_begin(xbroadcast<CT, X>& c) noexcept
```

Comment:
Should be `XTENSOR_CONSTEXPR_RETURN` instead of `constexpr` (as the generic declaration, the overload in `xscalar` has to be fixed)

Reply:
fixed.